runtime/doc/faq.txt

```diff
@@ -0,0 +1,701 @@
+*faq.txt*		 Frequently Asked Questions
+
+                            NVIM REFERENCE MANUAL
+
+==============================================================================
+1. General Questions                                 *faq* *faq-general-questions*
+
+
+WHERE SHOULD I PUT MY CONFIG (VIMRC)? ~
```

Comment:
I think we can fix the parser so that UPPERCASE lines support punctuation, then the trailing `~` won't be needed. This is fine for now though.

Reply:
(also modelines)